failing_tools.toml

```diff
@@ -0,0 +1 @@
+miri = "Testing"
```

Comment:
Such a line in a file called `failing_tools` indicates to me that miri fails to be successfully tested. But the comments in `bootstrap/failing_tools.rs` indicate something else. I think the naming here needs improvement.

Reply:
Indeed. I started out with a simple boolean, but found it too limiting. I renamed all the `failing_tools` things to `toolstate`.

In the future we might want to add more variants, like `Distribute`, in order to easily remove tools from distribution e.g. in beta and stable